package.json

```diff
@@ -42,10 +42,10 @@
     "sbt": "sbt universal:packageBin"
   },
   "dependencies": {
+    "@omega-edit/client": "^0.9.59",
```

Comment:
Make sure to update this to be `0.9.59` instead of `^0.9.59`. It is preferred to pin packages at a specific version. Having `^` basically allows the install to the version listed or anything newer.

Reply:
@scholarsmate This will still apply just with `0.9.60`

Reply:
Ah, good point!  Fixing...